client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -13,11 +13,12 @@ import Comments from '../stats-comments';
 import Followers from '../stats-followers';
 import PostingActivity from '../post-trends';
 import TodaysStats from '../stats-site-overview';
-import SiteOverviewPlaceholder from 'my-sites/stats/stats-overview-placeholder';
 import StatsModule from '../stats-module';
 import statsStrings from '../stats-strings';
 import MostPopular from 'my-sites/stats/most-popular';
 import LatestPostSummary from '../post-performance';
+import UpgradeNudge from 'my-sites/upgrade-nudge';
+import { CUSTOM_DOMAIN } from 'lib/plans/constants';
```

Comment:
Do you mean `FEATURE_CUSTOM_DOMAIN`? 

I'm seeing false being recorded for cta_feature:
![screen shot 2016-04-28 at 3 17 47 pm](https://cloud.githubusercontent.com/assets/1270189/14903206/6d747730-0d54-11e6-8f17-bfb7b5612348.png)